src/main/java/yandex/mail/pages/LoginPage.java

```diff
@@ -0,0 +1,19 @@
+package yandex.mail.pages;
+
+import static yandex.mail.Locators.LOG_IN;
+import static yandex.mail.Locators.USERNAME_INPUT;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage inputUsername() {
+        driver.findElement(USERNAME_INPUT).sendKeys("testuserte5t");
```

Comment:
sendKeys("testuserte5t"); - better to move this as class variable or add parameter to method